CHANGELOG.md

```diff
@@ -24,6 +27,7 @@ COMPONENTS:
 }
 ```
 * Fields in AirlockNotification event have changed without backward compatibility. If Airlock Notifier shared service is deployed, it needs to be re-deployed. Any other consumers of AirlockNotification event need to be updated. For more details, see [#2798](https://github.com/microsoft/AzureTRE/pull/2798)
+* The model for `reviewUserResources` in airlock requests has changed from being a list to a dictionary. A migration has been added to update your existing requests automatically; please make sure you run the migrations as part of updating your API and UI. [#2883](https://github.com/microsoft/AzureTRE/pull/2883)
```

Comment:
Should we mention that the in-flight requests will display an "UNKNOWN" user? (This is what will happen, right?)